examples/dreambooth/train_dreambooth_lora_sd3.py

```diff
@@ -1341,7 +1341,6 @@ def compute_text_embeddings(prompt, text_encoders, tokenizers):
     if not train_dataset.custom_instance_prompts:
         del tokenizers, text_encoders
         # Explicitly delete the objects as well, otherwise only the lists are deleted and the original references remain, preventing garbage collection
-        del tokenizer_one, tokenizer_two, tokenizer_three
```

Comment:
Should this not be done for https://github.com/huggingface/diffusers/blob/d2b10b1f4f92201a50bd44ff8bc2ea1b29d1b650/examples/dreambooth/train_dreambooth_sd3.py#L1372 as well?

Reply:
i think it's fine to leave the tokenisers loaded.

Reply:
done